site/topics/logic/combinational-logic.rst

```diff
@@ -506,13 +506,76 @@ Programmable Logic Arrays
 Functional Completeness
 =======================
 
+* As seen with PLAs, any boolean logic function can be implemented systematically
+
+    * In other words, any function can be physically created with configurations of AND, OR, and NOT gates
+
+
+* This means that the set of {AND, OR, NOT} gates is *Functionally complete*
```

Comment:
This fact is still mind impaling to me. The idea that everything machines do can be boiled down into NAND is bonkers. I get there's a lot of layers there, but still.